com.unity.render-pipelines.universal/Documentation~/writing-shaders-urp-basic-unlit-structure.md

```diff
@@ -0,0 +1,177 @@
+# URP unlit basic shader
+
+This example shows a basic URP-compatible shader. This shader fills the mesh shape with a color predefined in the shader code.
+
+To see the shader in action, copy and paste the following ShadeLab code into the Shader asset.
+
+```c++
+// This shader fills the mesh shape with a color predefined in the code.
+Shader "Example/URPUnlitShaderBasic"
+{
+    // The properties block of the shader. In this example this block is empty since 
```

Comment:
A shader asset defines a "Unity shader", not just a shader - it's a specific data structure.

Either change to "The properties block of the Unity shader." or just "The properties block."